kubernetes/start

```diff
@@ -30,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup || true
+./cleanup
```

Comment:
mainly this part https://github.com/kubic-project/velum/blob/master/kubernetes/cleanup#L6